workshop/machine-learning-in-Jupyter-notebook/README.md

```diff
@@ -2,41 +2,37 @@
 
 ## Build and Save a model
 
 For this part of the exercise we're going to use a Jupyter notebook to create the model. The Jupyter notebook is already included as an asset in the project you imported earlier.
 
->*Note: The Jupyter notebook included in the project has been cleared of output. If you would like to see the notebook that has already been completed with output: **Notebook with output**: [with-output/machinelearning-churn-sparkmlmodel-output.ipynb](../../notebooks/with-output/machinelearning-churn-sparkmlmodel-output.ipynb)*
+### Open the Jupyter Notebook
 
-Open the notebook:
+* Go the (☰) navigation menu and click on the *Projects* link and then click on your analytics project.
 
 * From the project overview page, *click* on the `Assets` tab to open the assets page where your project assets are stored and organized.
 
 * Scroll down to the `Notebooks` section of the page and *Click* on the pencil icon at the right of the `machinelearning-churn-sparkmlmodel` notebook.
 
-![Notebook Open](../.gitbook/assets/images/wml/wml-open-notebook.png)
-
-When the Jupyter notebook is loaded and the kernel is ready, we will be ready to start executing it in the next section.
+  ![Notebook Open](../.gitbook/assets/images/wml/wml-open-notebook.png)
 
-![Notebook loaded](../.gitbook/assets/images/wml/wml-3-notebook-loaded.png)
+* When the Jupyter notebook is loaded and the kernel is ready, we will be ready to start executing it in the next section.
 
-### Run the Jupyter notebook
+  ![Notebook loaded](../.gitbook/assets/images/wml/wml-3-notebook-loaded.png)
 
-Spend some time looking through the sections of the notebook to get an overview. A notebook is composed of text (markdown or heading) cells and code cells. The markdown cells provide comments on what the code is designed to do.
+### Run the Jupyter Notebook
 
-You will run cells individually by highlighting each cell, then either click the `Run` button at the top of the notebook or hitting the keyboard short cut to run the cell (Shift + Enter but can vary based on platform). While the cell is running, an asterisk (`[*]`) will show up to the left of the cell. When that cell has finished executing a sequential number will show up (i.e. `[17]`).
+Spend some time looking through the sections of the notebook to get an overview. A notebook is composed of text (markdown or heading) cells and code cells. The markdown cells provide comments on what the code is designed to do. You will run cells individually by highlighting each cell, then either click the `Run` button at the top of the notebook or hitting the keyboard short cut to run the cell (Shift + Enter but can vary based on platform). While the cell is running, an asterisk (`[*]`) will show up to the left of the cell. When that cell has finished executing a sequential number will show up (i.e. `[17]`).
```

Comment:
This looks like it should be upstream, and in all the workshops.
I like this phrasing.